tests/test_query.js

```diff
@@ -122,16 +122,27 @@ test('query() accepts URLSearchParams as input', async t => {
 })
 
 test('query() throws for duplicate keys', async t => {
-  const interceptor = nock('http://example.test')
-    .get('/')
-    .query({ foo: 'bar' })
+  const interceptor = nock('http://example.test').get('/?foo=bar')
 
   t.throws(
     () => {
       interceptor.query({ foo: 'baz' })
     },
     {
-      message: 'foo already defined as a query parameter',
+      message: 'Query parameters have already been already defined',
+    }
+  )
+})
+
+test('query() throws for invalid arguments', async t => {
```

Comment:
```suggestion
test('query() throws for invalid arguments', t => {
```

Will also need a `t.end()`.